.github/workflows/create-release.yml

```diff
@@ -175,12 +175,20 @@ jobs:
         cp install_sct.bat install_sct-${{ github.event.inputs.milestone_title }}_win.bat
         perl -pi -e 's/set git_ref=master/set git_ref=${{ github.event.inputs.milestone_title }}/' install_sct-${{ github.event.inputs.milestone_title }}_win.bat
 
+    - name: Create ${{ github.event.inputs.milestone_title }} Linux/macOS install scripts
+      # NB: `install_sct` works for both Linux and macOS platforms. So, here, we simply duplicate the script for both
+      # platforms (which will allow us to collect some rudimentary statistics + make our installation steps clearer).
+      run: |
+        cp install_sct install_sct-${{ github.event.inputs.milestone_title }}_linux.sh
+        perl -pi -e 's/SCT_GIT_REF="master"/SCT_GIT_REF="${{ github.event.inputs.milestone_title }}"/' install_sct-${{ github.event.inputs.milestone_title }}_linux.sh
+        cp install_sct-${{ github.event.inputs.milestone_title }}_linux.sh install_sct-${{ github.event.inputs.milestone_title }}_macos.sh
+
     - uses: ncipollo/release-action@v1
       name: Create release
       id: create_release
       with:
         tag: ${{ github.event.inputs.milestone_title }}
         token: ${{ secrets.GITHUB_TOKEN }}
         bodyFile: ".github/workflows/release-body.md"
-        artifacts: "install_sct-${{ github.event.inputs.milestone_title }}_win.bat"
+        artifacts: "install_sct-${{ github.event.inputs.milestone_title }}_win.bat,install_sct-${{ github.event.inputs.milestone_title }}_linux.sh,install_sct-${{ github.event.inputs.milestone_title }}_macos.sh"
```

Comment:
I wanted to split this line into multiple lines (for line length purposes) but I was having trouble wrapping my head around the intersection between:

1. The syntax for specifying multiple artifacts to `ncipollo/release-action`
    * The repo for this action states "This may be a single path or a comma delimited list of paths (or globs)", but provides no tangible examples of how a list should be expressed. (Do we use `[`/`]` brackets? Where do quotes go, if at all?)
    * Searching GitHub-wide for "ncipollo/release-action artifacts:" gave many examples, but going through a few of the 100+ pages, I couldn't find any that showed what "list of paths" looks like.
    * The closest I came was [this issue comment](https://github.com/ncipollo/release-action/issues/42#issuecomment-826150550), which offhandedly refers to the syntax `"string1,string2"` all within double-quotes, which is what I used here.
2. YAML multiline statements
    - Multiline YAML is so notoriously complex and error-prone that there's a whole site dedicated to it: https://yaml-multiline.info/
    - I figured that `>` would be what we would want, and so I tried [this syntax](https://github.com/joshuacwnewton/test-github-actions/commit/1853f59e2d2462c62c1c42ad3ff2f4f10b520f4c), but I got some [strange warnings](https://github.com/joshuacwnewton/test-github-actions/actions/runs/4307222314/jobs/7512024597#step:13:26), and the assets were not attached to the dummy 6.0 release.
    - I presume I maybe did something wrong in my "test github actions workflow", so I then got stuck in the weeds with the [PWDs of various GH Action steps](https://github.com/ncipollo/release-action/issues/42). Because this was rather time-consuming to test, I figured it would be safer to keep the string on a single line (at least for this first PR draft).